options/locale/locale_en-US.ini

```
@@ -1101,6 +1101,7 @@ pulls.required_status_check_failed = Some required checks were not successful.
 pulls.required_status_check_administrator = As an administrator, you may still merge this pull request.
 pulls.blocked_by_approvals = "This Pull Request doesn't have enough approvals yet. %d of %d approvals granted."
 pulls.blocked_by_rejection = "This Pull Request has changes requested by an official reviewer."
+pulls.blocked_by_outdated_branch = "This Pull Request is blocked by an outdated head branch."
```

```suggestion
pulls.blocked_by_outdated_branch = "This Pull Request is blocked because it's outdated."
```
I know it's less technical, but for many people (especially new git users) less confusing, IMHO.